packages/server/src/api/controllers/view/viewsV2.ts

```diff
@@ -125,6 +126,12 @@ export async function get(ctx: Ctx<void, ViewResponseEnriched>) {
   }
 }
 
+export async function fetch(ctx: Ctx<void, ViewFetchResponseEnriched>) {
```

Comment:
Why is this needed?

Reply:
As part of the public APIs we always offer an endpoint to list all resources of that type - this can then be searched by name. I've added this to the internal API so the public API can utilise it.